src/tests/ftest/pool/management_race.py

```diff
@@ -60,10 +57,10 @@ def del_recreate_query_and_list_pools(
                     try:
                         self.log.info("--(%d.4.%d.%d)--pool querying.. \n",
                                       thread_num, pool_num, test_num)
-                        query_result = self.get_dmg_command().pool_query(pool=label)
+                        self.get_dmg_command().pool_query(pool=label)
```

Comment:
```suggestion
                        label = self.pool[pool_num].label.value # Line 38
                        self.get_dmg_command().pool_query(pool=label)
```
On Line 38, it's better to obtain the label from the TestPool object than reconstructing it.